.vscode/settings.json

```diff
@@ -16,5 +11,10 @@
       },
       "editor.formatOnSave": true,
       "editor.defaultFormatter": "charliermarsh.ruff"
-    }
+    },
+    "python.testing.pytestArgs": [
```

Comment:
These changes are unrelated so could be in a separate PR?

Reply:
This can just go actually - I had to remove and re-add and VS code has just put in a different place in the file.